src/bandersnatch/mirror.py

```diff
@@ -596,36 +575,36 @@ def _save(self) -> None:
     def json_file(self, package_name: str) -> Path:
         return Path(self.webdir / "json" / package_name)
 
-    def json_pypi_symlink(self, package_name : str) -> Path:
+    def json_pypi_symlink(self, package_name: str) -> Path:
         return Path(self.webdir / "pypi" / package_name / "json")
 
     def simple_directory(self, package: Package) -> Path:
         if self.hash_index:
             return Path(self.webdir / "simple" / package.name[0] / package.name)
         return Path(self.webdir / "simple" / package.name)
 
-    def save_json_metadata(self, package_info: Dict) -> bool:
+    def save_json_metadata(self, package_info: Dict, name: str) -> bool:
```

Comment:
Probably what we should do here is just take a package as a parameter, and extract both from that.